iotdb-client/session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -3536,19 +3895,34 @@ public Session build() {
                 version);
         newSession.setEnableQueryRedirection(true);
         return newSession;
+      } else if (nodeUrls == null && useSSL) {
```

Comment:
please still use the same two branches as before, and don't have a separate if else branch for ssl